packages/frontend/src/components/DashboardFilter/FilterConfiguration/TileFilterConfiguration.tsx

```diff
@@ -203,97 +203,102 @@ const TileFilterConfiguration: FC<Props> = ({
-                                    value.invalidField
-                                        ? `The selected field ${value.invalidField} is not valid`
-                                        : 'No fields matching filter type'
-                                }
-                                position="left"
-                                disabled={
-                                    !value.disabled &&
-                                    value.invalidField === undefined
-                                }
-                            >
-                                <Box>
-                                    <Checkbox
-                                        size="xs"
-                                        fw={500}
-                                        disabled={value.disabled}
-                                        label={
-                                            <Flex align="center" gap="xxs">
-                                                <MantineIcon
-                                                    color="blue.8"
-                                                    icon={getChartIcon(
-                                                        value.tileChartKind,
-                                                    )}
-                                                />
-                                                <Text
-                                                    color={
-                                                        value.invalidField
-                                                            ? 'red'
-                                                            : undefined
-                                                    }
-                                                >
-                                                    {value.label}
-                                                </Text>
-                                            </Flex>
-                                        }
-                                        styles={{
-                                            label: {
-                                                paddingLeft: theme.spacing.xs,
-                                            },
-                                        }}
-                                        checked={value.checked}
-                                        onChange={(event) => {
-                                            onChange(
-                                                event.currentTarget.checked
-                                                    ? FilterActions.ADD
-                                                    : FilterActions.REMOVE,
-                                                value.tileUuid,
-                                            );
-                                        }}
-                                    />
-                                </Box>
-                            </Tooltip>
+            <Stack spacing="md">
```

Comment:
I only removed the parent wrapper `<Stack spacing="lg">` from here, to use it separately when no tabs are available (it is redundant with tabs)